ci/azure/linux.yml

```diff
@@ -81,7 +96,10 @@ jobs:
       displayName: 'Setup BigQuery credentials'
       condition: eq(variables['System.PullRequest.IsFork'], 'False')
 
-    - bash: make start PYTHON_VERSION=$PYTHON_VERSION BACKENDS="${BACKENDS}"
+    - bash: |
+        if [ -n "${BACKENDS}" ]; then
```

Comment:
Why the if statement?

Reply:
`make start` with `BACKENDS=""` starts all the backends instead of none of them because it essentially ends up running ```docker-compose up  ```  which starts all services (compared to something like ```docker-compose up mysql``` which would only start the mysql service)

Reply:
can you add a comment to this effect